manifests/base/jupyter-datascience-notebook-imagestream.yaml

```diff
@@ -36,7 +36,7 @@ spec:
             {"name": "Odh-Elyra", "version": "4.2"},
             {"name": "PyMongo", "version": "4.11"},
             {"name": "Pyodbc", "version": "5.2"},
-            {"name": "Codeflare-SDK", "version": "0.27"},
+            {"name": "Codeflare-SDK", "version": "0.28.1"},
```

Comment:
Something that was discussed yesterday in https://github.com/opendatahub-io/notebooks/pull/1085#issuecomment-2875933849. So we probably want to have `0.28` only there.

Reply:
Would this pick up the latest v0.28.1 if we just have 0.28? We need to release 0.28.1 as opposed to 0.28 as 0.28.0 has a bug

Reply:
This is only for presentation reasons on the UI. The image itself will include the `0.28.1` as it is defined on the pipfiles

Reply:
Ah I understand. Will update, ty!

Reply:
resolved in fc6c6dc